107/src/main/java/org/ehcache/jsr107/EhcacheCachingProvider.java

```diff
@@ -38,35 +39,32 @@
  */
 public class EhcacheCachingProvider implements CachingProvider {
 
+  private static final String DEFAULT_URI_STRING = "ehcache://default";
```

Comment:
I'd recommend a URI of something like `urn:X-ehcache:default-config`.

Unlike the current URI, this conforms to the relevant RFC and standards:
    * [BCP-115/RFC-4395 Guidelines and Registration Procedures for New URI Schemes](https://tools.ietf.org/html/rfc4395)
    * [BCP-66/RFC-3406 Uniform Resource Names (URN) Namespace Definition Mechanisms](https://tools.ietf.org/html/bcp66)
    * [STD-66/RFC-3986 Uniform Resource Identifier (URI): Generic Syntax](https://tools.ietf.org/html/rfc3986)
    * [RFC-2141 URN Syntax](https://tools.ietf.org/html/rfc2141)

We should stay away from using a new, unregistered scheme *and* avoid use of a hierachical designator (`//`) where none is intended (RFC-4395).